docs/overview/getting-started/virtual-machines.md

```diff
@@ -0,0 +1,94 @@
+# Virtual Machines
+
+Avalanche allows developers to implement their Dapp in the language of their choice using the 
+Virtual Machine (VM) framework. VMs define blockchains, and serve as application-level blueprints 
+for how blockchains are created.
+
+## Validators
+
+All Avalanche validators as members of the Avalanche primary network are required to run three VMs:
+
+- Coreth: Defines the Contract Chain (C-Chain); supports smart contracts written in Solidity.
+- Platform VM: Defines the Platform Chain (P-Chain); supports operations on staking and Subnets.
+- Avalanche VM: Defines the Exchange Chain (X-Chain); supports operations on Avalanche Native
+  Assets.
+
+Validators are able to install additional VMs on their node to validate additional Subnets in
+the Avalanche ecosystem. In exchange, validators receive staking rewards in the form of the Subnets'
+native gas tokens.
+
+To learn more about Subnets, click [here](/docs/subnets/README.md).
+
+## Building on Virtual Machines
+
+Previously, most blockchain ecosystems required developers to implement their applications in 
+the smart-contract language supported natively by the ecosystem. This causes pain for developers,
+as developers need to learn new technology stacks specific to each ecosystem they develop in.
+
+Developers with advanced use cases that required more customization were forced to start from 
```

Comment:
Rewrite this in the present tense to say what we provide more than what our competitors didn't

Reply:
It's basically all the same info

Reply:
Made this present tense